news/_posts/2018-01-09-graduation.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: news
+title: Lab member graduates
+author: jessicabartley
+category: news
+image: jb_angie_graduation
```

Comment:
You need the path and full filename for the picture (i.e., `/assets/images/news/jb_angie_graduation.jpg`)